lib/editor-settings.php

```diff
@@ -5,70 +5,6 @@
  * @package gutenberg
  */
 
-/**
- * Returns editor settings that are common to all editors:
- * post, site, widgets, and navigation.
- *
- * All these settings are already part of core,
- * see edit-form-blocks.php
- *
- * @return array Common editor settings.
- */
-function gutenberg_get_common_block_editor_settings() {
-	$max_upload_size = wp_max_upload_size();
-	if ( ! $max_upload_size ) {
-		$max_upload_size = 0;
-	}
-
-	$available_image_sizes = array();
-	// This filter is documented in wp-admin/includes/media.php.
-	$image_size_names = apply_filters(
-		'image_size_names_choose',
-		array(
-			'thumbnail' => __( 'Thumbnail', 'gutenberg' ),
-			'medium'    => __( 'Medium', 'gutenberg' ),
-			'large'     => __( 'Large', 'gutenberg' ),
-			'full'      => __( 'Full Size', 'gutenberg' ),
-		)
-	);
-	foreach ( $image_size_names as $image_size_slug => $image_size_name ) {
-		$available_image_sizes[] = array(
-			'slug' => $image_size_slug,
-			'name' => $image_size_name,
-		);
-	};
-
-	$settings = array(
-		'__unstableEnableFullSiteEditingBlocks' => gutenberg_is_fse_theme(),
```

Comment:
I removed it when moving to `get_default_block_editor_settings` because it doesn't belong to WP core and it's handled in a filter.